src/mods/common/analyzer/update/bpy.types.mod.rst

```diff
@@ -47,6 +47,45 @@
       :type seq: collections.abc.Sequence[bool] | collections.abc.Sequence[int] | collections.abc.Sequence[float] | typing_extensions.Buffer
       :mod-option arg seq: skip-refine
 
+.. class:: bpy_prop_collection_idprop
+
+   :generic-types: _GenericType1
+
+   .. method:: values()
+
+      :rtype: list[_GenericType1 | None]
+      :mod-option rtype: skip-refine
+
+   .. method:: find(key)
+
+      :type key: str (never None)
+
+   .. method:: get(key, default=None)
+
+      :type key: str (never None)
```

Comment:
Maybe `never None` can be added by option field `:option key: never none`.
Some for others.

Could you check it?

Reply:
The type annotation seems correct to me, `key` only accepts `str`s.
![image](https://github.com/user-attachments/assets/7158d949-fb43-49b2-ad1f-3f958fe66a3b)

Reply:
This is for `bpy_prop_collection`, I did not test with `bpy_prop_collection_idprop`.

Reply:
Yeah, the annotation itself is fine to me.
This is just a optimization idea.

My suggestion is that we can skip-refine if we add `:option key: never none` field.
In this case, we can process faster compared to this.

Reply:
Would this improve performance significantly or is this just the more elegant solution?

Reply:
Updated mods to skip-refine, there other remaining but they are in other classes than `bpy_prop_collection(_idprop)`.

Reply:
Thanks. It looks fine.